framework/test/unit/modules/interoperability/mainchain/commands/recover_message.spec.ts

```diff
@@ -584,25 +591,53 @@ describe('MessageRecoveryCommand', () => {
 					),
 				};
 
+				const recoveredCCM = await command['_applyRecovery'](ctx);
```

Comment:
due to this, the expectation in line 597 will be met, even though it was not triggered from the function you are testing, right? Alternatively, you could create a copy of `ctx`, change the status of `ccm` to `CCM_STATUS_CODE_RECOVERED` and swap `ccm.sendingChainID` and `ccm.receivingChainID`.